### PR TITLE
xz 5.6.2

### PR DESCRIPTION
**Pull request by johnalanwoods**
- [ x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/x/xz.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@ class Xz < Formula
...
   # The archive.org mirror below needs to be manually created at `archive.org`.
-  url "https://github.com/tukaani-project/xz/releases/download/v5.4.6/xz-5.4.6.tar.gz"
-  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.4.6.tar.gz"
-  mirror "https://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  mirror "http://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  sha256 "aeba3e03bf8140ddedf62a0a367158340520f6b384f75ca6045ccc6c0d43fd5c"
+  url "https://github.com/tukaani-project/xz/releases/download/v5.4.6/xz-5.6.2.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.2.tar.gz"
+  mirror "https://archive.org/download/xz-5.4.6/xz-5.6.2.tar.gz"
+  mirror "http://archive.org/download/xz-5.4.6/xz-5.6.2.tar.gz"
```
Comment (SMillerDev) [inline Formula/x/xz.rb:8]:
```suggestion
  url "https://github.com/tukaani-project/xz/releases/download/v5.6.2/xz-5.6.2.tar.gz"
  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.2.tar.gz"
  mirror "https://archive.org/download/xz-5.6.2/xz-5.6.2.tar.gz"
  mirror "http://archive.org/download/xz-5.6.2/xz-5.6.2.tar.gz"
```

**Comment by SMillerDev** [comment]
The test is failing so someone needs to figure out what is wrong and how to fix that.

**Comment by Omoeba** [comment]
Can anyone take a look at it to see what's wrong?

**Comment by Omoeba** [comment]
Any ideas why it took so absurdly long?

**Comment by SMillerDev** [comment]
> Any ideas why it took so absurdly long? I'm not too familiar with the testing process, but 1508 minutes do not look right to the slightest.

Because a loooooot of things depend on `xz`

**Comment by Omoeba** [comment]
Is the CI failing related to the patch mentioned in the [release](https://github.com/tukaani-project/xz/releases/tag/v5.6.2) or is it something unrelated?

**Comment by Omoeba** [comment]
Is there anything else needed for this to be merged?

**Comment by johnalanwoods** [comment]
Given that we see ~250K downloads a month and we're still on a release that didn't actively have the backdoor removed, this should be prioritised.

I don't know the Brew CI/CD process, but it looks like the errors are related to cancellation mostly.

**Comment by p-linnane** [comment]
Kicking off a new run. Will check back in periodically to see if there are hard blockers to merging, or if it's just stuff we can follow up on.

**Comment by p-linnane** [comment]
The `zstd` failures are a regression in brew. Barring any other failures, I'll merge this once CI is done.

**Review by p-linnane — APPROVED** [review]

**Comment by p-linnane** [comment]
Regression in brew fixed in https://github.com/Homebrew/brew/pull/17780 by @issyl0.

**Pull request merged**